template/.gitignore.jinja2

```diff
@@ -7,6 +7,8 @@ bin/
 *.tmp.owl
 *.tmp.json
 
+.github/token
```

Comment:
you settled on this location as default for the token? I am wondering, since `.github` is actually checked in git, whether the name of the token file should start with period, i.e. `.token`, to indicate that we kinda don't want it checked in? Maybe its stupid because it will make using the infinitely harder for Windows users, who cant create files starting with a period without jumping through major hoops

Reply:
(what I am saying: you are probably right in calling it `token`

Reply:
Yeah, since we already have a `.github` directory for, well, GitHub-related stuff, I didn’t see any point in imagining yet another hidden file or directory.

As for calling it `.github/.token`: I wouldn’t see the point either, the `.github` directory is already hidden. And the file is listed in `.gitignore` anyway, so users can only commit it if they force it to be committed – and if they do that, well, they’re free to do stupid things if they absolutely want to.

Reply:
Ok, makes sense! To make this easier for windows, would we consider `token.txt`?

Reply:
Sure. Note however that this won’t be of any use for now, since I have not updated the `run.bat` script for Windows (and I have no plans of doing it, since I cannot test it), so the feature is for Unix-users only for now.

I am sorry for Windows users, but some of them (at least one!) will have to step up and contribute if they want the same level of features than Unix-like users…